### PR TITLE
Use Azure pipelines for CI

### LINKED ISSUE #1031: Migrate to azure pipelines for CI

### PR DESCRIPTION
**Pull request by touilleMan**
fix #1031

### CODE REVIEW

File: .azure-pipelines.yml [hunk 1]
```diff
@@ -0,0 +1,298 @@
...
+trigger:
+  branches:
+    include:
+    - '*'
+  tags:
+    include:
+    - '*'
+
+variables:
+  python.version: '3.7'
```
Comment (vxgmichel) [inline .azure-pipelines.yml]: Bumping to 3.7? Does snap support that?

File: parsec/core/gui/ui/__init__.py [hunk 2]
```diff
@@ -0,0 +1 @@
+# Parsec Cloud (https://parsec.cloud) Copyright (c) AGPLv3 2019 Scille SAS
```
Comment (vxgmichel) [inline parsec/core/gui/ui/__init__.py]: Was that really necessary ? :thinking:

Comment (touilleMan) [reply parsec/core/gui/ui/__init__.py]: pre-commit did this, not me ;-)

File: .azure-pipelines.yml (hunk 1 above)
Comment (touilleMan) [reply .azure-pipelines.yml]: snap build with 3.6, so we should stick to 3.6 !

**Pull request merged**